RELEASE_NOTES.md

```diff
@@ -7,6 +7,14 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current major release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.9.1 (2016-9-26)
+
+### Chef Server
+
+* Fix CHEFDK_GECODE_PATH for latest ChefDK
+* Fixed typo
+* Updated openssl
```

Comment:
If you take a look in PRIOR_RELEASE_NOTES there's a good example of how to break this into a separate ``Security Updates`` section with links to the relevant CVEs